[2017/11/27 18:49:18] :: [pool-1-thread-1] :: Base path for the test = C:\New Selenium\Salesforce_QA_Automation\ADT_Resources\<br><br>[2017/11/27 18:49:18] :: [pool-1-thread-1] :: Current Thread Name for the test is "pool-1-thread-1"<br><br>[2017/11/27 18:49:26] :: [pool-1-thread-1] :: The user has launched 'https://moveinc.my.salesforce.com<br><br>[2017/11/27 18:49:39] :: [pool-1-thread-1] :: arindom.das@move.com has successfully loggedin<br><br>[2017/11/27 18:49:40] :: [pool-1-thread-1] :: The user has creating ZUORA PRODUCTION CONNECTIONScenario<br><br>[2017/11/27 18:50:13] :: [pool-1-thread-1] :: Exception occurred. null<br><br>